tests/cloudpickle_test.py

```diff
@@ -508,6 +509,21 @@ def test_load_dynamic_module_in_grandchild_process(self):
             if os.path.exists(child_process_module_file):
                 os.unlink(child_process_module_file)
 
+    def test_correct_globals_import(self):
+        import math
+
```

Comment:
Could you please also define a useless function named "unwanted_function" and another named "nested_function".

Reply:
This is non-regression test: the goal is to make assertions of the expected working of the library to make sure that this is not broken by future changes we cannot anticipate.

Reply:
So yeah just add an `unwanted_function` in the local scope of the test, that is the same scope as the one where `my_small_function` is defined so as to ensure that it is not included in the pickle byte string (to keep it as lightweight as possible).